### ISSUE TITLE
[updatecli] track `mirrorbits` CLI version

### ISSUE THREAD

**Issue by dduportal** [jenkins-infra/jenkins-infra#3645]
### What feature do you want to see added?

#3642 introduced the `mirrorbits` CLI installation. Its version is pinned to `v0.5.1` in a hieradata YAML file.

We should track the version of mirrorbits with updatecli and update it if a new one is released.

### Upstream changes

https://github.com/jenkins-infra/jenkins-infra/pull/3642 (part of https://github.com/jenkins-infra/helpdesk/issues/2649)

**Comment by jayfranco999** [comment]
Added updatecli manifest to track the latest version of mirrorbits CLI from the GH repo - https://github.com/jenkins-infra/jenkins-infra/pull/3646

Tested locally and successful (rolled-back mirrorbits version v.0.5.0 to check the solution)

```
updatecli diff --config ./updatecli/weekly.d/mirrorbits.yaml --values ./updatecli/values.yaml


+++++++++++
+ PREPARE +
+++++++++++

Loading Pipeline "./updatecli/weekly.d/mirrorbits.yaml"

SCM repository retrieved: 1


++++++++++++++++++
+ AUTO DISCOVERY +
++++++++++++++++++



++++++++++++
+ PIPELINE +
++++++++++++



#############################
# BUMP `MIRRORBITS` VERSION #
#############################


SOURCES
=======

lastReleaseVersion
------------------
Searching for version matching pattern "latest"
✔ GitHub release version "v0.5.1" found matching pattern "latest" of kind "latest"


CHANGELOG:
----------

Release published on the 2018-11-23 12:25:27 +0000 UTC at the url https://github.com/etix/mirrorbits/releases/tag/v0.5.1

## v0.5.1

### ENHANCEMENTS

- Sort the mirrors by the last state date in the list command

### BUGFIXES

- Regression: mirrors were not able to transition between up and down states


TARGETS
========

updateHieradataVersion
----------------------

**Dry Run enabled**

⚠ - change detected:
        * key "$.profile::mirrorbits::mirrorbits_version" should be updated from "v0.5.0" to "v0.5.1", in file "hieradata/common.yaml"


ACTIONS
========


=============================

REPORTS:



⚠ Bump `mirrorbits` version:
        Source:
                ✔ [lastReleaseVersion] Get the latest `mirrorbits` version
        Target:
                ⚠ [updateHieradataVersion] Update the `mirrorbits` version in the hieradata/common.yaml file


Run Summary
===========
Pipeline(s) run:
  * Changed:    1
  * Failed:     0
  * Skipped:    0
  * Succeeded:  0
  * Total:      1

**Comment by jayfranco999** [comment]
Thanks @smerle33 for troubleshooting my GH Token blockers :)

**Comment by smerle33** [comment]
closed by #3646
